domain/build.gradle

```diff
@@ -28,5 +28,6 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.jetbrains:annotations:19.0.0'
```

Comment:
Add comment
//Domain layer should have no dependencies. that said, Java doesn't support Nullability so this one dependecy deemed ok

Reply:
Leaving this conversation open for future possible discussion around this matter